### PR TITLE
[Windows, 4.0] Open a new console window for the editor instances.

### LINKED ISSUE #54076: Godot will not startup from project manager if the default terminal application is set to Windows Terminal in Windows 11
___

***Bugsquad note:** This issue has been confirmed several times already. No need to confirm it further.*

___

### Godot version

4.0.dev(fad7034e8b)

### System information

Windows 11

### Issue description

Godot will not startup from project manager if set the default terminal software to Windows Terminal in Windows 11, it will close immediately

### Steps to reproduce

1. Change the default terminal application from cmd to Windows Terminal in Windows Terminal Settings.
2. Try to open a project in Godot's project manager
3. Godot window close immediately like a crash

### Minimal reproduction project

_No response_

### PR DESCRIPTION
**Pull request by bruvzg**
Fixes #54076

Workaround for the new Windows terminal issue.

### CODE REVIEW

File: platform/windows/os_windows.cpp [hunk 1]
```diff
@@ -484,6 +484,9 @@ Error OS_Windows::create_process(const String &p_path, const List<String> &p_arg
 #ifndef DEBUG_ENABLED
 	dwCreationFlags |= CREATE_NO_WINDOW;
 #endif
+	if (p_path == get_executable_path() && GetConsoleWindow() != NULL) {
```
Comment (akien-mga) [inline platform/windows/os_windows.cpp:487]: This should be `nullptr` for consistency with the rest of the codebase.

**Review by akien-mga — APPROVED** [review]

**Review by mhilbrunner — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by sairam4123** [comment]
Could this be cherry-picked to 3.x?

**Comment by mhilbrunner** [comment]
Yup, its been merged for 3.x already: https://github.com/godotengine/godot/pull/55925

**Pull request merged**